### DISCUSSION TITLE
mfsu 编译node_modules中的包如何支持flow以及extension，alias配置支持

### DISCUSSION THREAD

**Discussion by wkeylin** [umijs/umi#10081] [Q&A]
目前在用umi 做react-native-web 移动端的项目支持，但是react-native-\*很多包都是flow, 以及多平台文件，*.ios.js, *.web.js,*.android.js等以及需要将react-native的引用替换成react-native-web，mfsu编译时好像没提供处理类似场景的配置
<img width="1079" alt="image" src="https://user-images.githubusercontent.com/21221921/208046355-fdf40379-7728-4831-a34e-285c6a222b38.png">

**Comment by wkeylin** [comment]
关闭mfsu整个项目是可以正常运行的，但是想要快点的热更新以及热启动

**Reply by fz6m** [reply]
这个错误可能和你现在有的 ts 版本不支持 `import { type Type } from 'xxx'` 这种语法有关，确认下你项目的 ts 版本是不是最新的?

**Reply by wkeylin** [reply]
ts版本没问题的`4.9.3`, 这个就是包里的js文件，flow语法

**Reply by fz6m** [reply]
尝试一下这个吧：

```ts
// .umirc.ts

export default {
  extraBabelPlugins: [
    // flow babel plugin ...
  ],
  extraBabelPresets: [
    // flow babel preset ...
  ],
  mfsu: { strategy: 'normal' }
}
```
